README.rst

```diff
@@ -55,7 +55,7 @@ solutions. The hope is for Matrix to act as the building blocks for a new
 generation of fully open and interoperable messaging and VoIP apps for the
 internet.
 
-Synapse is a Matrix "homeserver" implementation developed by the matrix.org core 
```

Comment:
Thanks. I need to figure out how to configure PyCharm to not do this when it automatically wraps a line.

Reply:
I don't think this trailing space was your doing, tbh, and it's not exactly the end of the world anyway. But my editor likes to strip them...